org.learningequality.Kolibri.Plugin.Explore.json

```diff
@@ -0,0 +1,27 @@
+{
+  "id": "org.learningequality.Kolibri.Plugin.Explore",
+  "branch": "1.0",
```

Comment:
```suggestion
```

Reply:
Remove this will make the master to make it "stable" branch by default? For this plugin the version is `1.0`, so I think this should match the `org.learningequality.Kolibri` manifest: https://github.com/flathub/org.learningequality.Kolibri/pull/21/files#diff-c2e15187e8c009872b3048917e454df9fecdf4e591e865f98a5df7411dc28e87R27

Reply:
We usually avoid versioning app-specific extensions unless you predict users will have multiple branches of an extension installed.

Reply:
Ok, I've updated the PR to the original repository to remove the version so this can work with that: https://github.com/flathub/org.learningequality.Kolibri/pull/21/commits/7e15f721dda9bae32c39e2671accab5ff868e76b